src/lib/src/header/header.component.css

```diff
@@ -1,55 +1,20 @@
 /*
  * PRX header component
  */
-header {
+:host {
   position: fixed;
   top: 0;
   right: 0;
   left: 0;
   width: 100%;
   max-width: 1200px;
   margin: 0 auto;
-  height: 50px;
   z-index: 200;
   border-bottom: 1px solid #e6e6e6;
-  background: #fff;
-}
-.contents {
-  max-width: 1060px;
-  margin-right: auto;
-  margin-left: auto;
-  padding-right: 15px;
-  padding-left: 15px;
-}
-@media screen and (min-width: 768px) {
-  .contents {
-    padding-right: 30px;
-    padding-left: 30px;
-  }
-}
-a.drawer {
-  position: absolute;
```

Comment:
none of this `.drawer` business was being used. I'm about to do some work on responsive design and would rather just start fresh with what we are using.

Reply:
+1